firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/CrashlyticsRegistrar.java

```diff
@@ -24,7 +24,7 @@
 import com.google.firebase.inject.Deferred;
 import com.google.firebase.installations.FirebaseInstallationsApi;
 import com.google.firebase.platforminfo.LibraryVersionComponent;
-import com.google.firebase.sessions.FirebaseSessions;
```

Comment:
@mrober, let you know this is still in AQS change https://github.com/firebase/firebase-android-sdk/blob/master/firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/CrashlyticsRegistrar.java, in fact it is not a merge conflict.

Reply:
Your change lgtm